app/models/lookup_key.rb

```diff
@@ -158,6 +158,11 @@ def path_elements
     end
   end
 
+  def overriden_on_host?(host)
```

Comment:
Past participle is overridden (double d), I think there is no need to specify 'on_host' as it can be overridden on host groups (perhaps other places in the future).